app/src/main/java/com/futurice/freesound/feature/home/HomeFragment.kt

```diff
@@ -53,7 +53,7 @@ class HomeFragment : BindingBaseFragment<HomeFragmentComponent>() {
                 .observeOn(schedulerProvider.ui())
                 .subscribe({
                                picasso.load(it)
-                                   .transform(PicassoTransformations.circular())
+                                   .transform(circular())
```

Comment:
I'm not sure that this namespace change is for the better.

Reply:
The change is because `circular` is in "no" class. And I dont think that using it like `com.futurice.freesound.feature.images.circular())` here would benefit it

Reply:
That being the case, I'll rename `circular` to `circularTransformation` to make it clearer.